### PR TITLE
[bash/en] Addition to bash

### PR DESCRIPTION
**Pull request by LOZORD**
- [x] PR touches only one file (or a set of logically related files with similar changes made)
- [x] Content changes are aimed at *intermediate to experienced programmers* (this is a poor format for explaining fundamental programming concepts)
- [x] YAML Frontmatter formatted according to [CONTRIBUTING.md](https://github.com/adambard/learnxinyminutes-docs/blob/master/CONTRIBUTING.markdown)
  - [x] Seriously, look at it now. Watch for quotes and double-check field names.

Incorporates information from #1807 and #1465. I attempted to make the information a little less basic and moved the "prologue" I had initially into the bash code itself.

### CODE REVIEW

**Comment by LOZORD** [comment]
@vendethiel , can you [squash this](https://github.com/blog/2141-squash-your-commits)? Otherwise I can attempt it 😬

File: bash.html.markdown [hunk 1]
```diff
@@ -90,6 +95,21 @@ echo "Number of arguments passed to script: $#"
...
+# `pwd` stands for "print working directory".
+# We can also use the builtin variable `$PWD`.
+# Observer that the following are equivalent:
+echo "I'm in $(pwd)" # execs `pwd` and interpolates output
+echo "I'm in $PWD" # interpolates the variable
+
+# If you get too much output in your terminal, or from a script, the command
+# `clear` clears your screen
+clear
+# Ctrl-L also works for clearning output
```
Comment (vendethiel) [inline bash.html.markdown]: Typo "clearning"

File: bash.html.markdown [hunk 2]
```diff
@@ -138,12 +158,37 @@ ls
...
 
+# Use `cat` to print files to stdout:
+cat file.txt
+
+# We can also read the file using `cat`:
+Contents=$(cat file.txt)
+echo "START OF FILE\n$Contents\nEND OF FILE"
+
+# Use `cp` to copy files or directories from one place to another.
+# `cp` creates NEW version of the sources,
```
Comment (vendethiel) [inline bash.html.markdown]: Creates **a**

**Comment by LOZORD** [comment]
Thanks for catching those 👍

**Comment by vendethiel** [comment]
thanks for your PR 👍

**Pull request merged**